debian/control

```diff
@@ -16,6 +16,7 @@ Description: A common package which contains files which are shared by several C
 Package: cloudstack-management
 Architecture: all
 Depends: ${misc:Depends}, ${python:Depends}, openjdk-8-jre-headless | java8-runtime-headless | java8-runtime, cloudstack-common (= ${source:Version}), tomcat6 | tomcat7, sudo, jsvc, python-mysql.connector, libmysql-java, augeas-tools, mysql-client, adduser, bzip2, ipmitool, lsb-release
+Recommends: init-system-helpers
```

Comment:
Can we force it to a specific version, default it builds with init-system-helpers (>= 1.18~); if we can add >= 1.14 to support ubuntu 14.04?